CIPs/0007.md

```diff
@@ -0,0 +1,65 @@
+# CIP [0007]: Affordable Gas Downtime Slasher
+
+- Date: 2020-05-05
+- Author: @gastonponti (with a lot of feedback and ideas from @m-chrzan)
+- Status: Implemented
```

Comment:
This needs to be changed to Draft status

Reply:
Just to make sure we are on the same page, this is implemented and being deployed with contract release 1. Is the status flipped once the contract is finally deployed?

Reply:
Exactly